pkg/kube/client/client.go

```diff
@@ -37,7 +37,7 @@ func (c client) CreateOrUpdate(obj runtime.Object) error {
 		}
 		return err
 	}
-	return c.Update(context.TODO(), obj)
+	return c.Update(context.TODO(), objCopy)
```

Comment:
👍

Reply:
This solution does not work of course. It always updates the object that was just retrieved by the API!